fonts/stylesheets/fonts.less

```diff
@@ -193,6 +193,11 @@
 .font ( 'ia-writer-mono', bold, normal, 'ia-writer-mono/ia-writer-mono-bold.woff' );
 .font ( 'ia-writer-mono', bold, italic, 'ia-writer-mono/ia-writer-mono-bold-italic.woff' );
 
+.font ( 'intel-one-mono', normal, normal, 'intel-one-mono/intelone-mono-font-family-regular.woff2' );
```

Comment:
I would go with

“intel-one-mono/intel-one-mono.woff2”
“intel-one-mono/intel-one-mono-italic.woff2”
“intel-one-mono/intel-one-mono-bold.woff2”

..etc

Reply:
Thanks for that latest change. Please run the build script again to update the stylesheet.

Reply:
😳 Sorry that I forgot that. It has been done.